spec/models/ticket_request_spec.rb

```diff
@@ -404,5 +404,20 @@
         end
       end
     end
+
+    describe 'guests_array' do
```

Comment:
Please DRY this as follows: make an rspec `subject` from `ticket_request.guests_present`:

**Then you can use the following syntax to shorten the spec:**

```ruby
subject { ticket_request.guests_present }

its(:class) {is_expected.to be(Array)} 

its(:size) { is_expectd.to eql(2) }

describe 'Append a blank guest' do
  before { subject << '' }

  its(:size) { is_expectd.to eql(2) }
end
#```